stdlib/src/builtin/string.mojo

```diff
@@ -1974,8 +1959,12 @@ struct String(
         Returns:
             A copy of the string with no trailing whitespaces.
         """
-        # TODO: should use self.__iter__ and self.isspace()
         var r_idx = len(self)
+        # TODO: should use this once llvm intrinsics can be used at comp time
```

Comment:
Here and in the other TODOs: Could you please find the (or open a new) ticket for tracking this, and reference it here?

Reply:
Added the issue number #933